src/tilemaker.cpp

```diff
@@ -278,14 +284,13 @@ int main(int argc, char* argv[]) {
 			} else if (srcZ > config.startZoom) {
 				cout << "Mapsplit tiles (zoom " << srcZ << ") can't write data at zoom level " << config.startZoom << endl;
 			}
+
 			cout << "Reading tile " << srcZ << ": " << srcX << "," << srcY << " (" << (run+1) << "/" << runs << ")" << endl;
 			vector<char> pbf = mapsplitFile.readTile(srcZ,srcX,tmsY);
-			// Write to a temp file and read back in - can we do this better via a stringstream or similar?
```

Comment:
I also removed this temporary file, but I did not test this. Can you test this ? It should be OK llike this